src/Psalm/Type/Atomic/TNamedObject.php

```diff
@@ -27,17 +27,24 @@ class TNamedObject extends Atomic
      */
     public $was_static = false;
 
+    /**
+     * Whether or not this type can represent a child of the class named in $value
+     * @var bool
+     */
+    public $can_be_inheritance = false;
```

Comment:
I think the name can be improved. How about something along the lines of `includes_descendants`, or `definite_class`?

Reply:
done